hazelcast/src/test/java/com/hazelcast/journal/AbstractEventJournalBasicTest.java

```diff
@@ -73,7 +72,6 @@
     private int partitionId;
     private TruePredicate<EJ_TYPE> TRUE_PREDICATE = new TruePredicate<EJ_TYPE>();
     private Function<EJ_TYPE, EJ_TYPE> IDENTITY_FUNCTION = new IdentityFunction<EJ_TYPE>();
-    private boolean loadAllPublishesAdded = false;
 
     private void init() {
```

Comment:
We can add the `@Before` annotation back here and remove the `init()` calls from the test cases. I introduced explicit `init()` calls only to make the load behavior parameterized, see https://github.com/hazelcast/hazelcast/pull/13674/files#r214365582.

Reply:
lately seen this one, also addressed this.